### PR TITLE
Create new art using p5.js

### LINKED ISSUE #67: Create new art using p5.js
### Pre-requisities
Familiarity with p5.js

### Context
We are preparing for genesis 3 of Blockalizer. For reference:
- [this](https://opensea.io/assets/ethereum/0xbe440a8cd331cadb9ab611c6000072beb74b963a/780) is how genesis 1 looked
- [this](https://opensea.io/assets/ethereum/0xbe440a8cd331cadb9ab611c6000072beb74b963a/1564) is how genesis 2 looked
Genesis 2 also introduced animations that won't be the case for genesis 3

### What
Update art for genesis 3 to that it looks as follows:
Screen Shot 2023-03-07 at 3 19 44 PM

The filled circle should represent 1. And unfilled circle represents 0. Every shape is determined by the block number which is provided as input. Additionally, for aesthetics, some shapes are drawn as half circles. For that, the orientation can be determined via randomization. For example, choose a random number between `[0,1,2]` if 0 then full unfilled circle, if 1 then half unfilled circle facing right, if 2 then half unfilled circle facing left. Feel free to choose your own randomization algorithm. You have flexibility over this component as long as the final image looks visually appealing.  Also, feel free to use generic colors. The art is generated using p5.js library.

### Hints
The art style is determined by [this function](https://github.com/Alpha-Labs-Global/blockalizer/blob/master/src/helper/sketch.ts#L39). You have to create a new sketch style [here](https://github.com/Alpha-Labs-Global/blockalizer/tree/master/src/art-styles). It's invoked from [here](https://github.com/Alpha-Labs-Global/blockalizer/blob/master/src/components/Art.tsx#L106). Note how `BlockInfo` and `BlockNumber` are passed as inputs. You'd have to do something similar. And use that to determine the shape. Consider [this style](https://github.com/Alpha-Labs-Global/blockalizer/blob/master/src/art-styles/colored-triangles.ts) as an inspiration.

### PR DESCRIPTION
**Pull request by cuddle-dev**

### CODE REVIEW

File: src/pages/Playground.tsx [hunk 1]
```diff
@@ -545,7 +545,7 @@ const Playground: React.FC<ComponentProps> = (props: ComponentProps) => {
               animate={animate}
               setAnimate={setAnimate}
               premium={premium}
-              disableMint={disableMint || blockUI}
```
Comment (uneeb123) [inline src/pages/Playground.tsx]: Why remove `blockUI`?

File: src/scripts/make_palette.js [hunk 2]
```diff
@@ -25,11 +25,15 @@ function hexToRgb(hex) {
 // var sunshine = ["#FFDAA8", "#FFAA2C", "#FF9218", "#311400"];
 // var noir = ["#E6E6E6", "#979797", "#8A8A8A", "#070707"];
 // var allColors = [alpine, tidal, autumn];
-var tropicana = ["#FFC634", "#7534FF", "#FF34D2"];
```
Comment (uneeb123) [inline src/scripts/make_palette.js:28]: cool I see you were able to decipher this? This is just a script I did to generate the palette quickly

File: src/art-styles/circle_sketch.ts [hunk 3]
```diff
@@ -0,0 +1,480 @@
...
+//   "https://maroon-petite-shrew-493.mypinata.cloud/ipfs/Qmc1UUU1dkfnKQNAtuZEq5QEDNpwAj4DR5N8eP2VFVxxgk",
+// ];
+
+const paper_links: Array<string> = [
+  "https://blockalizer-animation-template.s3.us-east-2.amazonaws.com/paper1.webp",
+  "https://blockalizer-animation-template.s3.us-east-2.amazonaws.com/paper2.webp",
+  "https://blockalizer-animation-template.s3.us-east-2.amazonaws.com/paper3.webp",
+];
+
+export class CircleSketch extends GenericSketch {
```
Comment (uneeb123) [inline src/art-styles/circle_sketch.ts:13]: This file contains a whole bunch of things that I think you aren't using.

File: src/art-styles/circle_sketch.ts [hunk 4]
```diff
@@ -0,0 +1,480 @@
...
+    this.paperIndex = opts.paperIndex;
+  }
+
+  setup(canvasParentRef: Element) {
+    this.p5
+      .createCanvas(this.canvasWidth, this.canvasHeight)
+      .parent(canvasParentRef);
+
+    this.p5.frameRate(120); // highest possible framerate
+    this.p5.loadImage(
```
Comment (uneeb123) [inline src/art-styles/circle_sketch.ts]: why are u even loading image?

Comment (cuddle-dev) [reply src/art-styles/circle_sketch.ts]: Its not needed actually, will push cleanup commits soon

File: src/art-styles/circle_sketch.ts (hunk 3 above)
Comment (cuddle-dev) [reply src/art-styles/circle_sketch.ts:13]: yes will clean up we aren't using

File: src/pages/Playground.tsx (hunk 1 above)
Comment (cuddle-dev) [reply src/pages/Playground.tsx]: I think I used hardcoded value to see the UI working. it should be reverted

**Pull request merged**